src/languages/cs.js

```diff
@@ -1,25 +1,37 @@
+  }
+
   var KEYWORDS = {
     keyword:
       // Normal keywords.
       'abstract as base bool break byte case catch char checked const continue decimal ' +
       'default delegate do double else enum event explicit extern finally fixed float ' +
-      'for foreach goto if implicit in int interface internal is lock long ' +
+      'for foreach goto if implicit in int interface internal is lock long nameof new ' +
       'object operator out override params private protected public readonly ref sbyte ' +
       'sealed short sizeof stackalloc static string struct switch this try typeof ' +
       'uint ulong unchecked unsafe ushort using virtual void volatile while ' +
-      'nameof ' +
       // Contextual keywords.
       'add alias ascending async await by descending dynamic equals from get global group into join ' +
       'let on orderby partial remove select set value var where yield',
     literal:
-      'null false true'
+      'null false true',
+    built_in:
+      SYSTEM.classes,
+    class:
```

Comment:
`class` is reserved for definitions of user classes, not built-in class names. We clump together all the built-ins regardless of their nature.